### ISSUE TITLE
use-onblur-not-onchange overly strict

### ISSUE THREAD

**Issue by erin-doyle** [romeovs/react-a11y#11]
I'm getting a `use-onblur-not-onchange` error for a number of elements that have both `onchange` and `onblur` handlers and rightfully so.  Looking at the respective rule in the [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/049136246a429302399bc46ec4a67b776a1ea87c/src/rules/use-onblur-not-onchange.js) it looks like it is checking for when an element only has `onchange` it fails otherwise if it has both `onchange` and `onblur` it passes.  I wonder if this rules needs to be updated to do the same.
